spec/features/admin_deletes_book_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+feature 'Admin deletes a book', js: true do
+  scenario 'successfully' do
+    book = create(:book)
+    admin = create(:user, :admin)
+
+    visit root_path(as: admin)
+
+    expect(page).to have_content(t('books.delete.link'))
```

Comment:
I don't think you need this since `delete_book` will fail if there is no delete link on the page